### DISCUSSION TITLE
Typing issue: Unable to write specific kanji, e.g 教 for "Teach"

### DISCUSSION THREAD

**Discussion by zDEFz** [google/mozc#852] [Q&A]
## Category of the typing issue
Choose one of them (delete rest of them)
4. Others

## Issues
Write issues to the following table. (It's in the markdown format)

| input [e.g.ゆうひ] | expected [e.g. 夕日] | actual [e.g. ユウヒ] |
| ------------ | ------------ | ------------|
| (きょう) | (write here) | (optional, write here) |

## Version or commit-id
[e.g. Mozc-2.28.4960.100+24.11.oss or d50a8b9ae28c4fba265f734b38bc5ae392fe4d25]
You can get the version string by converting "Version" or "ばーじょん".
mozc 2.29.5268.102-1

## Additional context

I enter きょう then press tab and cant find the kanji. 
How am I normally supposed to type it?

**Comment by yukawa** [comment]
> I enter きょう then press tab and cant find the kanji.

Try pressing the space key multiple times after entering きょう.

**Comment by zDEFz** [comment]
> > I enter きょう then press tab and cant find the kanji.
> 
> Try pressing the space key multiple times after entering きょう.

I need further support. I essentially try to write 値 which means direct, straight, honest, but after looking through 80 recommendations I doubt its in it.

I can get 直 but I can't get 値 somehow

**Reply by phoepsilonix** [reply]
> I need further support. I essentially try to write 値 which means direct, straight, honest, but after looking through 80 recommendations I doubt its in it.
> 
> I can get 直 but I can't get 値 somehow

In my environment,
Type in "きょう" and some of the candidates that come up include "教" (teaching) and "今日" (today) or today's date is a candidate.

直: なお　ちょく すぐ じき　癖が直る（くせがなおる）
正直:しょうじき honest
直接:ちょくせつ direct
率直:そっちょく direct
直接的：ちょくせつてき direct
直線:ちょくせん straight line
直球:ちょっきゅう straight ball,straight

値:あたい　ね　価値（かち）　値する（あたいする） value
値段:ねだん price
値札:ねふだ price tag

教える:おしえる
教室:きょうしつ classroom
教科書:きょうかしょ textbook
教壇に立つ:きょうだんにたつ　stand on a podium

**Comment by zDEFz** [comment]
Why does yours appear so different? I use fcitx5-mozc choku/chiki isn't the same as chi/atai

https://up.tail.ws/specific-kanji-fcitx5-mozc.mp4

Update: Writing あたい worked. Then I get 値 https://www.nihongomaster.com/japanese/dictionary/kanji/1831/%E5%80%A4

**Comment by zDEFz** [comment]
I was told I use the wrong 'kanji font', that I should not get a chinese character out of mozc.

they told me that is japanese 直 whereas 値 is chinese. Is this correct?

It appears this is related to region/regional format

**Reply by tats-u** [reply]
Both can be Japanese or Chinese. It can be ambiguous whether a given short kanji characters sequence is Japanese or Chinese.

**Comment by tats-u** [comment]
You have to learn how to read Japanese kanji characters much more first before using Japanese IME including those other than Mozc/Google Japanese Input.

**Reply by zDEFz** [reply]
> You have to learn how to read Japanese kanji characters much more first before using Japanese IME including those other than Mozc/Google Japanese Input.

Nah, am fine, am learning my way :1st_place_medal:
